system/include/emscripten/emscripten.h

```diff
@@ -262,6 +262,7 @@ double emscripten_asm_const_double(const char *code, ...);
 #if __EMSCRIPTEN__
 void emscripten_sleep(unsigned int ms);
 void emscripten_sleep_with_yield(unsigned int ms);
+void emscripten_block_on(char *identifier);
```

Comment:
We should have documentation about this function in `site\source\docs\api_reference\emscripten.h.rst` and `site\source\docs\porting\emterpreter.rst`, as this is public. Also, is the pattern always that `emscripten_block_on` is called in C code, and the counterpart to that is `Module.notifyAbout`, which is called from JS code?